### PR TITLE
scikit_wrappers.py bug fix

### PR DESCRIPTION
**Pull request by zionsteiner**
fit_classifier() didn't return anything if svc penalty was specified and train_size // nb_classes >= 5 and train_size >= 50

### CODE REVIEW

File: scikit_wrappers.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ def fit_classifier(self, features, y):
             else numpy.inf,
             gamma='scale'
         )
-        if train_size // nb_classes < 5 or train_size < 50:
+        if train_size // nb_classes < 5 or train_size < 50 or self.penalty:
```
Comment (White-Link) [inline scikit_wrappers.py]:
```suggestion
        if train_size // nb_classes < 5 or train_size < 50 or self.penalty is not None:
```

File: scikit_wrappers.py [hunk 2]
```diff
@@ -169,7 +169,7 @@ def fit_classifier(self, features, y):
...
             return self.classifier.fit(features, y)
         else:
             if self.penalty is None:
```
Comment (White-Link) [inline scikit_wrappers.py]:
```suggestion
```

**Review by White-Link — CHANGES_REQUESTED** [review]
Hi, thanks for reporting this issue!

There's a corner when `self.penalty` is zero, in which case the method will not return anything as well. If this is fixed your pull request will be ready to be merged.

**Comment by zionsteiner** [comment]
Should be fixed now

**Review by White-Link — APPROVED** [review]

**Pull request merged**